docs/components/evaluation.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: 🔬 Evaluation
+---
+
+## Overview
+
+We provide out-of-the-box evaluation methods for your datasets. You can use them to evaluate your models and compare them with other models.
+
+Currently, we provide the following evaluation methods:
+
+<CardGroup cols={3}>
+    <Card title="Context Relevancy" href="#context_relevancy"></Card>
+    <Card title="Answer Relevancy" href="#answer_relevancy"></Card>
+    <Card title="Groundedness" href="#groundedness"></Card>
+    <Card title="Custom" href="#custom"></Card>
+</CardGroup>
+
```

Comment:
Can you please define the three metrics here on docs briefly on how it is calculated?